### ISSUE TITLE
Display of active cases vs time

### ISSUE THREAD

**Issue by nikohansen** [covid19-dash/covid-dashboard#84]
Thanks for sharing this, very nice project! I'd like to make a few small suggestions on the time evolution graphs:

- it seems that the active cases over time should be normalized by the number people (country size). I found the closed issue #3, but it seems not to be effective yet? The number of cases in itself is a meaningless object, in particular when compared across countries. The _effect_ of a single case is always relative to the number of health care professionals who can take care of it.

- Using `log` as default scale: if I click a new country, the scale switches back to linear and displaying these data on linear scale is to me close to useless. Of course, it is good enough if the scale does not switch back.

- I'd prefer if the two figures, world map and time evolution, were on top of each other rather than side-by-side, such that they can be displayed larger on the screen.

I made a similar effort that you can find [here ![](http://www.cmap.polytechnique.fr/~nikolaus.hansen/new-cases-link-2.png)](http://www.cmap.polytechnique.fr/~nikolaus.hansen/covid19-figure2.html)

with my rationales given [here](https://scienceandrandomthoughts.wordpress.com/2020/03/24/covid-19-related-trend-of-new-cases-per-day/) but I'd also be happy to join forces. I did it because I couldn't find a single source plotting new cases per day on a log scale for a variety of countries.

**Comment by poldrack** [comment]
I agree that either log should be the default, or the scale choice should be sticky when one changes the country selection

**Comment by GaelVaroquaux** [comment]
We have now implemented the normalization for the time series.

With regards to the log vs linear scale: I agree that the resetting is a bug (it's dash / javascript, and I do not know how to fix it). However, given that our target audience is the layperson, I would rather have linear scale by default: people do not have intuitions about exponential growth (including decision makers). I think that the best would be to fix the bug that reset. Such a contribution would be incredibly helpful.

With regards to you last point: try blowing up the fonts for the page (ctrl-+ under firefox): the layout will adapt.

Cheers!

**Comment by nikohansen** [comment]
> try blowing up the fonts for the page (ctrl-+ under firefox): the layout will adapt.

I (obviously) tried. This is what I get in Chrome:

<img width="1343" alt="Screen Shot 2020-03-25 at 20 29 42" src="https://user-images.githubusercontent.com/7316439/77577716-8bc21d80-6ed7-11ea-9967-6d0c244a115c.png">

The _figure_ I am interested in now occupies a far too small amount of my screen, to my taste. On the positive side, the legend is readable ;-) If I make it larger, the legend flips in the next row and becomes huge and difficult to access. On the positive side, then the figure occupies the full screen.

**Comment by GaelVaroquaux** [comment]
Thanks for the screenshot. You may be hitting a rendering bug that makes the table too large.

I suspect that you are on a high-DPI screen. Can you tell me your DPI, as detected by your browser (visible be visiting the following page https://www.infobyip.com/detectmonitordpi.php ), your screen width, and the br

**Comment by nikohansen** [comment]
Rightly so suspected. The page says `192 x 192` (on a recent 15.4" 2880 x 1800 Macbook pro).

**Comment by GaelVaroquaux** [comment]
OK, I'm working on addressing this in #88

(you can check out the link to circle CI artifacts, in the status badges of the PR, to visualize a rendered webpage).

**Comment by nikohansen** [comment]
Fixed the font size in the ~legend~ table, but the table width is unchanged (i.e. its widths is not adapted to the font size).

**Comment by GaelVaroquaux** [comment]
How about the version of #88 ?

**Comment by nikohansen** [comment]
No fix of width with a101234383692477f09ff8e4d51bade9760814e1, as far as I can tell (sorry, was distracted).
